pdal/pipeline.py

```diff
@@ -107,6 +110,9 @@ def get_meshio(self, idx: int) -> Optional[Mesh]:
         )
 
     def _get_json(self) -> str:
```

Comment:
should we add a deprecation warning when calling this method explicitly?

Reply:
I don't think so. It was explicitly not part of the public API and it was internal to the library (and was only out there for one release).